src/picongpu/include/simulationControl/MySimulation.hpp

```diff
@@ -480,6 +483,28 @@ class MySimulation : public SimulationHelper<simDim>
     {
         namespace nvfct = PMacc::nvidia::functors;
 
+        typedef typename PMacc::particles::traits::FilterByIdentifier
+        <
+            VectorAllSpecies,
+            momentumPrev1
+        >::type VectorSpeciesWithMementumPrev1;
+
+        /* copy attribute momentum to momentumPrev1 */
```

Comment:
IMO this is not the right place for this comment, the order of the plugin calls and the runOneStep should commented in `PMacc::SimulationHelper`.
The order inside run one step is visible by the calls itself.